badges/badges.css

```diff
@@ -0,0 +1,65 @@
+@import "../avatar/avatar.css";
+
+*{
+    --online-color: #108a2c;
+    --dnd-color: #a51d2b;
+    --offline-color: #8f8f8f;
+    --away-color: #c9a711;
+    --icon-badge-color: #c40c1e;
+}
+
+.avatar{
+    border: none;
+}
+.badge{
+    border-radius: 50%;
+    position: absolute;
+    background-color: var(--icon-badge-color);
+    text-align: center;
+}
+
+.badge-container{
+    position: relative;
+}
+
+.badge-online{
+    bottom: 1.2rem;
+    right: 1rem;
+    width: 2rem;
+    height: 2rem;
+    background-color: var(--online-color);
+}
+.badge-dnd{
+    bottom: 1rem;
+    right: 6px;
+    width: 1.75rem;
+    height: 1.75rem;   
+    background-color: var(--dnd-color);   
+}
```

Comment:
I have a suggestion here to have a single class with width, height and positioning for badges in general. And then to have separate classes with colors. Thereafter you can incorporate these classes depending on the usecases.